lang/languages.xml

```diff
@@ -38,4 +38,5 @@ List of available languages for your wallet's seed:
         <language display_name="עברית" locale="he_HE" wallet_language="English" flag="/lang/flags/israel.png" qs="none"/>
         <language display_name="한국어" locale="ko_KO" wallet_language="English" flag="/lang/flags/south_korea.png" qs="none"/>
         <language display_name="Română" locale="ro_RO" wallet_language="English" flag="/lang/flags/romania.png" qs="none"/>
-</languages>
+        <language display_name="Dansk" locale="da_DK" wallet-language="English" flag="/Laang/flags/denmark.png" qs="none"/>
```

Comment:
`/Laang/flags/denmark.png` should be `lang/flags/denmark.png`

Reply:
WOops. Will fix immediately. :-)